sdks/java/core/src/main/java/org/apache/beam/sdk/util/state/StateContexts.java

```diff
@@ -24,7 +24,7 @@
 import org.apache.beam.sdk.values.PCollectionView;
 
 /**
- * Factory that produces {@link StateContext} based on different inputs.
+ * Static methods for working with {@link StateContext}.
```

Comment:
How does this comment change relate to compatibility?

Also, isn't that static methods for creating {@link StateContext}, not "working with"?